cmd/run/tracer/tracer.go

```diff
@@ -105,89 +55,60 @@ func initTunnel(ctx context.Context, tunnelID uuid.UUID, endpoint string) (_ *we
 		}
 		return nil, err
 	}
-
-	defer func() {
```

Comment:
I can't say I have full context on this code but I love the fact that we're deleting so much :P

Reply:
red diffs best diffs